evap/static/scss/components/_notebook.scss

```diff
@@ -43,42 +43,44 @@
         flex-grow: 1;
     }
 
-    @media screen and (min-aspect-ratio: $notebook-break) {
-        .notebook-container {
-            width: 25vw;
-            padding-left: $spacer;
-            padding-bottom: $spacer;
-            padding-top: $spacer;
-        }
 
-        .notebook-card {
-            height: calc((100vh - #{$navbar-brand-height}) - (#{$footer-height} + ((#{$spacer} * 1.5) * 5))); // subtract 3 spacer with padding
-        }
+    .notebook-container {
+        width: 80vw; //approximation to be similar to evap-Content width
+        z-index: $zindex-fixed;
+        padding: $spacer;
     }
 
-    @media screen and (max-aspect-ratio: $notebook-break) {
-        .notebook-container {
-            width: 100vw;
-            z-index: $zindex-fixed;
-            padding: $spacer;
-        }
-
         .notebook-card {
             height: calc(45vh - (#{$spacer} * 1.5) * 2); // subtract spacer with padding
             min-height: 43.5vh; // approximation for size needed in smallest window (50vh - (spacer + leeway))
         }
+
+
+    @media screen and (min-aspect-ratio: $notebook-break), (min-width: 1400px) {//1400px chosen by expert judgement
+    .notebook-container {
+        width: 25vw;
+        padding-left: $spacer;
+        padding-bottom: $spacer;
+        padding-top: $spacer;
+        padding-right: 0;
+        z-index: none;
```

Comment:
Is `none` a valid value for `z-index`? It's not listed on [the MDN docs](https://developer.mozilla.org/en-US/docs/Web/CSS/Reference/Properties/z-index#values)...